web/themes/contrib/civictheme/includes/form.inc

```diff
@@ -45,6 +45,13 @@ function civictheme_preprocess_input(array &$variables): void {
   if (isset($variables['attributes']['#civictheme_theme'])) {
     unset($variables['attributes']['#civictheme_theme']);
   }
+
+  // Fix htmlentity in submit button text.
```

Comment:
This approach is not correct: the change needs to happen in the Twig because the underlying issue related to Twig sanitising output.